pkg/ring/kv/dynamodb/client_test.go

```diff
@@ -228,6 +228,40 @@ func Test_WatchKey_UpdateStale(t *testing.T) {
 	})
 }
 
+func Test_CAS_UpdateStale(t *testing.T) {
+	ddbMock := NewDynamodbClientMock()
+	codecMock := &CodecMock{}
+	descMock := &DescMock{}
+	descMockResult := &DescMock{id: "result"}
```

Comment:
We set the `id` field here but where do we read it? I don't see how it is used

Reply:
I was kust playing with the objects to make sure they are correct. I want to cache the result from CAS and not the info received in the initial query. A diff obj is enough for that. I can remove it